test/functional/tests/conftest.py

```diff
@@ -169,8 +169,7 @@ def base_prepare(item):
 
         for disk in TestRun.dut.disks:
             disk.umount_all_partitions()
-            if not create_partition_table(disk, PartitionTable.gpt):
-                raise Exception(f"Failed to remove partitions from {disk}")
+            create_partition_table(disk)
```

Comment:
Why did you change it? You could add it as separate commit with explanation in commit message

Reply:
I'll make it part of update-test-framework commit